rules/autoconsent/onlyFans.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "onlyFans.com",
+  "prehideSelectors": ["body > div"],
```

Comment:
This is too generic and hides the entire content on lots of sites. Please make it more specific, or just remove it for this rule.

Reply:
I used the same selectors as detectCMP - If it does not work I'll remove it. Thanks